tests/Jellyfin.Api.Tests/Jellyfin.Api.Tests.csproj

```diff
@@ -41,4 +41,10 @@
     <CodeAnalysisRuleSet>../jellyfin-tests.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
 
+  <ItemGroup>
+    <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
+      <_Parameter1>Jellyfin.Api.Tests</_Parameter1>
+    </AssemblyAttribute>
+  </ItemGroup>
+
```

Comment:
```suggestion
```
Don't think this is needed

Reply:
We do, the function is internal

Reply:
Yeah I know but expose the internals from `Jellyfin.Api.Tests` to itself doesn't make any sense

Reply:
Oh wait...how did this happen